pkg/policy/policy.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	BundleVersion    = 1 // Latest released MAJOR version for trivy-checks
-	BundleRepository = "ghcr.io/aquasecurity/trivy-checks"
+	BundleRepository = "mirror.gcr.io/aquasec/trivy-checks"
```

Comment:
We also use GHCR as a secondary for other databases. Just in case mirror.gcr.io is down, Trivy falls back into GHCR.

Reply:
Totally agree with @knqyf263, also the `--checks-bundle-repository` flag should also provide fallback functionality just like the `--db-repository` flag. Right now we can only specify a single checks repository with this option, it would we great if multiple checks repo can be specified using this option so that if one repo fails second one can be used as a fallback.

Reply:
We're discussing it.

Reply:
Thanks

Reply:
Yes we should. Track https://github.com/aquasecurity/trivy/issues/7960

Reply:
Thanks